src/Components/Alert/Components/Filters/ShowMore.tsx

```diff
@@ -0,0 +1,39 @@
+import { Clickable, Column, Text } from "@artsy/palette"
+import { Children, isValidElement, useState } from "react"
+import * as React from "react"
+
+interface ShowMoreProps {
+  initial?: number
+  expanded?: boolean
+}
+
+export const INITIAL_ITEMS_TO_SHOW = 6
+
+export const ShowMore: React.FC<ShowMoreProps> = ({
```

Comment:
I have created a new `ShowMore` for the two-column `GridColumn` design.

Reply:
[Palette's ShowMore ](https://palette.artsy.net/elements/showmore/#gatsby-focus-wrapper)component looks a bit different and cannot be used with `GridColumn` at the moment. Therefore, I thought it would be better to create a custom component instead of modifying the component from Palette to work in different places.